src/history.ts

```diff
@@ -69,7 +73,7 @@ class HistoryController {
      *
      * @param {string} id ID of the historyEntry
      */
-    promoteToActive(id: string): boolean {
+    promoteToActive(id: string) {
```

Comment:
Because the compiler already knows about the exact type from the return statement.
Should I add a rule to force these?

Reply:
I think it is generally a good thing to force return types for at least public methods so that any change to the return type creates "local" errors only (should result in easier-to-understand compile errors). 

Since this project didn't have any typing before, you can leave it as it is for now. :)

Reply:
Forcing them now [`5718f98` (#154)](https://github.com/ifl0w/RandomWallpaperGnome3/pull/154/commits/5718f982e7fbe3b090cf36f2cfaaa7a5ec782d2d)